src/multivariate/mvnormal.jl

```diff
@@ -54,6 +54,11 @@ end
 
 _pdf!(r::AbstractArray, d::AbstractMvNormal, x::AbstractMatrix) = exp!(_logpdf!(r, d, x))
 
+# Sampling with designated rng
+rand(rng::AbstractRNG, d::AbstractMvNormal) = _rand!(rng, d, Array{eltype(d)}(length(d)))
```

Comment:
I'd use `Vector{eltype(d)}(length(d))` since it's clearer from a cursory glance that it's 1-dimensional, plus it's easier for the compiler, which prefers not to have to determine how many dimensions there are itself.